Future/inst/shiny/meal app/app_for_meal/app.R

```diff
@@ -0,0 +1,283 @@
+library(shiny)
+
+
+header <- dashboardHeader(title = "Future")
+
+sidebar <- dashboardSidebar()
```

Comment:
until we have only one tab, maybe we can remove sidebar?

Reply:
we can't - tagAssert

Reply:
https://stackoverflow.com/questions/35170065/hide-sidebar-in-default-in-shinydashboard

Reply:
done